build.sbt

```diff
@@ -585,43 +472,90 @@ def isJdiJar(file: File): Boolean = {
   if (!System.getProperty("java.specification.version").startsWith("1.")) false
   else file.getAbsolutePath.contains(SbtJdiTools.JavaTools.getAbsolutePath.toString)
 }
+val publishJsonModuleSettings = List(
+  publishM2Configuration := publishM2Configuration.value.withOverwrite(true),
+  publishLocalConfiguration := publishLocalConfiguration.value.withOverwrite(true),
+  // We compile in both so that the maven integration can be tested locally
+  publishLocal := publishLocal.dependsOn(publishM2).value
+)
+
+lazy val integrationUtils211 = project
+  .in(integrations / "utils")
+  .settings(scalafixSettings)
+  .settings(
+    scalaVersion := Dependencies.Scala211Version,
+    libraryDependencies += Dependencies.bloopConfig,
+    target := (file(
+      "integrations"
+    ) / "utils" / "target" / "utils-2.11").getAbsoluteFile
+  )
+
+lazy val integrationUtils212 = project
+  .in(integrations / "utils")
+  .settings(scalafixSettings)
+  .settings(
+    scalaVersion := Dependencies.Scala212Version,
+    libraryDependencies += Dependencies.bloopConfig,
+    target := (file(
+      "integrations"
+    ) / "utils" / "target" / "utils-2.12").getAbsoluteFile
+  )
+
+lazy val integrationUtils213 = project
+  .in(integrations / "utils")
+  .settings(scalafixSettings)
+  .settings(
+    scalaVersion := Dependencies.Scala213Version,
+    libraryDependencies += Dependencies.bloopConfig,
+    target := (file(
+      "integrations"
+    ) / "utils" / "target" / "utils-2.13").getAbsoluteFile
+  )
```

Comment:
So it's a little annoying that I had to define 3 like this, but that's it's sort of due to how the build is defined and not using something like sbt-projectmatrix. Either way this seems to do the trick... All for one file that we need cross built 😆 .

Reply:
Should we introduce `sbt-projectmatrix` then? Would it simplify the build?

Reply:
We should yes, but I'd rather not do it in this pr. Because if we do it here we also have to all the other modules that are defined this way as well. I know in https://github.com/scalacenter/bloop/issues/1680 switching to `sbt-ci-release` is mentioned. I think maybe doing that migration and switching to `sbt-projectmatrix` at the same time would be a good idea. If we all agree, after I get this merged in I can try to tackle that.

Reply:
Why is `crossTarget` not working here? It seems that `target/utils-x.yz` is reproducing what `crossTarget` does?

Reply:
> Why is crossTarget not working here? It seems that target/utils-x.yz is reproducing what crossTarget does?

I believe `crossTarget` is only for Scala-version dependent targets: like class files, zinc cache...

Here we want the whole target to be independent. From sbt's point of view, those projects are completely separate and they can't share the same target directory.